src/frontend/src/routes/(new-styling)/+page.svelte

```diff
@@ -63,89 +61,86 @@
+                      <UserIcon size="1.25rem" />
+                    {/if}
+                  </Avatar>
+                  <div class="flex flex-col text-left text-sm">
+                    <div class="font-semibold">
+                      {identity.name ?? identity.identityNumber}
                     </div>
-                  </ButtonCard>
-                </li>
-              {/each}
-            </ul>
-            <ButtonCard
-              onclick={() => (isAuthDialogOpen = true)}
-              disabled={nonNullish(authLastUsedFlow.authenticatingIdentity)}
-            >
-              <FeaturedIcon size="sm">
-                <PlusIcon size="1.25rem" />
-              </FeaturedIcon>
-              <span>Use another identity</span>
-            </ButtonCard>
-          </div>
-          {#if isAuthDialogOpen}
-            <AuthDialog
+                    <div class="text-text-tertiary" aria-hidden="true">
+                      {"passkey" in identity.authMethod ? "Passkey" : "Google"}
+                    </div>
+                  </div>
+                </ButtonCard>
+              </li>
+            {/each}
+          </ul>
+          <ButtonCard
+            onclick={() => (isAuthDialogOpen = true)}
+            disabled={nonNullish(authLastUsedFlow.authenticatingIdentity)}
+          >
+            <FeaturedIcon size="sm">
+              <PlusIcon size="1.25rem" />
+            </FeaturedIcon>
+            <span>Use another identity</span>
+          </ButtonCard>
+        </div>
+        {#if isAuthDialogOpen}
+          <Dialog
+            onClose={() => (isAuthDialogOpen = false)}
+            showCloseButton={!isAuthenticating}
+            closeOnOutsideClick={!isAuthenticating}
+          >
+            <AuthWizard
+              bind:isAuthenticating
               {onSignIn}
               {onSignUp}
-              title="Use another identity"
-              subtitle="Choose method"
-              onClose={() => (isAuthDialogOpen = false)}
-            />
-          {/if}
-        {:else}
-          <PickAuthenticationMethod
-            setupOrUseExistingPasskey={authFlow.setupOrUseExistingPasskey}
-            continueWithGoogle={authFlow.continueWithGoogle}
-          />
-          {#if authFlow.view !== "chooseMethod"}
-            <Dialog onClose={() => (authFlow.view = "chooseMethod")}>
-              {#if authFlow.view === "setupOrUseExistingPasskey"}
-                <SetupOrUseExistingPasskey
-                  setupNew={authFlow.setupNewPasskey}
-                  useExisting={authFlow.continueWithExistingPasskey}
-                />
-              {:else if authFlow.view === "setupNewPasskey"}
-                <CreatePasskey create={authFlow.createPasskey} />
-              {/if}
-            </Dialog>
-          {/if}
+              onError={(error) => {
+                isAuthDialogOpen = false;
+                handleError(error);
+              }}
+              withinDialog
+            >
+              <h1
```

Comment:
The children of `AuthWizard` is the header at the top, right?

Would it make it clearer if we used a snippet "header" instead of children?